config.schema.yml

```diff
@@ -379,7 +379,7 @@ properties:
                                     type: "boolean"
                                 realnameFormat:
                                     type: "string"
-                                    enum: ["mxid","reverse-mxid"]
+                                    pattern: "^template:|^mxid$|reverse-mxid$"
```

Comment:
I think that regex wouldn't match `template:TEMPLATE`.

Reply:
it does actually; after the review suggestion from　@jaller94 it should now match anything that starts with `template:`